396: [Frontend] user profile page

NyahVW opened the pull request.

mvdrputt reviewed (changes requested):
Possible to enter invalid email. When applying change, email doesn't update.

mvdrputt reviewed (changes requested):
Header has 3 unused css styles

mvdrputt reviewed (changes requested):
![image](https://github.com/user-attachments/assets/57fcecb0-1bab-4779-b0ef-b146402272da)
![image](https://github.com/user-attachments/assets/e61443c9-26a2-4e62-91ad-502212b0f8f8)

mvdrputt commented:
It is still possible to enter invalid e-mail, luckily it doesn't change de password in the backend

NyahVW commented:
> It is still possible to enter invalid e-mail, luckily it doesn't change de password in the backend

fixed this

> ![image](https://private-user-images.githubusercontent.com/121724694/444228607-57fcecb0-1bab-4779-b0ef-b146402272da.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDc0MTM4NjIsIm5iZiI6MTc0NzQxMzU2MiwicGF0aCI6Ii8xMjE3MjQ2OTQvNDQ0MjI4NjA3LTU3ZmNlY2IwLTFiYWItNDc3OS1iMGVmLWIxNDY0MDIyNzJkYS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwNTE2JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDUxNlQxNjM5MjJaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0xNDczYTNjMThhNDIwMDA0MDQxNTc4YWUzMWZiY2YxNjhjZDU2MDljMmY3ODJiNmNlNDk3MDIwNGU3NWU4MDY4JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.-Nsn5W82wlGES8G_RwO2ey2P8uqE2gWHdUXlKlGglfM) ![image](https://private-user-images.githubusercontent.com/121724694/444228664-e61443c9-26a2-4e62-91ad-502212b0f8f8.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDc0MTM4NjIsIm5iZiI6MTc0NzQxMzU2MiwicGF0aCI6Ii8xMjE3MjQ2OTQvNDQ0MjI4NjY0LWU2MTQ0M2M5LTI2YTItNGU2Mi05MWFkLTUwMjIxMmIwZjhmOC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwNTE2JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDUxNlQxNjM5MjJaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0yNjg2ODExMGYyYTE0MmU5NDc0MTU2M2I0YWMyOTNmOGNmZGEwNzliZmU2MDkzNDJiYzQ3MWUxMzg1MDBlZGQzJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.Pt0ZFfU50_sL6hKULpvex_-qVNidHW5SsC8RAjsx9nA)

also fixed this
